src/test/java/com/onwelo/tinvalidator/validator/CZRValidatorTest.java

```diff
@@ -0,0 +1,77 @@
+package com.onwelo.tinvalidator.validator;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+@DisplayName("Czech Republic TIN should")
+class CZRValidatorTest {
+
+    @ParameterizedTest
+    @DisplayName("contain ten digits")
+    @CsvSource(value = {
+            "1111111111, true",
+            "111111111, true",
+            "1, false",
+            "99999, false"
+    })
+    void tinShouldContainTenOrNineDigits(final String tin, final boolean expectedResult){
+        Validator v = new CZRValidator();
+        Assertions.assertEquals(expectedResult, v.matchRegex(tin));
+    }
+
+    @ParameterizedTest
+    @DisplayName("be able to contain optional slash (/) between 6th and 7th character")
+    @CsvSource(value = {
+            "111111/1111, true",
+            "1111111111, true",
+            "111111/111, true",
+            "111111111, true",
+            "11/1111111, false",
+            "1, false",
+            "99999, false"
+    })
+    void tinShouldBeAbleToContainOptionalSlashBetween6thAnd7thCharacter(final String tin, final boolean expectedResult){
+        Validator v = new CZRValidator();
+        Assertions.assertEquals(expectedResult, v.matchRegex(tin));
+    }
+
+    @ParameterizedTest
+    @DisplayName("not contain letters")
+    @CsvSource(value = {
+            "1111111111, true",
+            "111111111, true",
```

Comment:
test, that is checking letter containment should have fixed proper length (length is checked in above one)